sentry_sdk/utils.py

```diff
@@ -867,8 +867,12 @@ def realign_remark(remark):
         value=rv, metadata={"len": rv_original_length, "rem": rv_remarks}
     )
 
-
 try:
+    if not PY2 and sys.version_info < (3, 7):
+        try:
```

Comment:
Instead of checking `sys.modules` at some completely other place, can we export constants here that tell you whether context vars work with asyncio, and use them in integrations?